Ctl/tmplvars.yaml

```diff
@@ -0,0 +1,7 @@
+project_name: "ixctl"
+service_home: "/src/ixctl"
+virtual_env: "venv"
+database_name: "ixctl"
+database_user: "ixctl"
+database_password: "devPASSWORD"
```

Comment:
where does this end up in the templating process? is this just to populate the example env file?

Reply:
Looks like this gets propagated into the docker-compose file, i think the password should come from .env and not through templating, since we dont want to expose the password in the repository.

@egfrank